client/resources/js/main.js

```diff
@@ -36,17 +36,30 @@
                 },
                 mapTypeId: google.maps.MapTypeId.ROADMAP
             };
-            
+
             var map = new google.maps.Map(document.getElementById("map"), options);	// create new map using settings above
             // var marker = new google.maps.Marker({						// place a marker at our lat/long
             //     position:	coords,
             //     map:		map
             // });
 
            // /client/resources/icons/Assault.svg
-            
-        
-            let locations = [ 
+           // var contentString = '<div id="content">'+
```

Comment:
Will these commented line be usefull in the near future?
I suggest we remove them if no.